huaweicloud/resource_huaweicloud_cce_node_pool.go

```diff
@@ -166,6 +167,12 @@ func ResourceCCENodePool() *schema.Resource {
 						},
 					}},
 			},
+			"tags": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				ForceNew: true,
```

Comment:
according to the [API document](https://support.huaweicloud.com/api-cce/cce_02_0356.html#cce_02_0356__request_V3NodeSpec), `tags` can be updated. Please have a check.

Reply:
you can using `tagsSchema()` to replace it.  @Jason-Zhang9309

Reply:
fixed